### DISCUSSION TITLE
Custom Columns cannot display in Search Results and Search Filters Webparts

### DISCUSSION THREAD

**Discussion by JenniferWang45** [microsoft-search/pnp-modern-search#2101] [Q&A]
**Version used**
4.5.4

**Describe the bug**
I added custom columns in the PNP Search Results and Filters Webparts, but none of the custom columns display. 
I configured the PNP Search web parts following the instructions, mapped the custom columns to managed properties(refinablestring). I also reindexed the site a few times, the columns are still not showing. 

**To Reproduce**
1. Created custom column
3. Mapped columns to refinablestring
4. Reindex Site
5. Add PNP Search result and Filters web parts
6. Selected the refinablestrings Properties during the Search Results configuration
7. Added the refinablestrings in the "Manage Columns" section of Search Results
8. Connected Search Filter to Search Results webpart
9. Added the refinablestrings in the Search Filter configurtaion
10. None of the custom columns show in the Search Results and Search Filter. Only able to see the default site columns.  

**Expected behavior**
The Custom Columns should display in the Search Results and Search Filters Web part

**Screenshots**
![Screen Shot 2022-05-12 at 12 27 04 PM](https://user-images.githubusercontent.com/105457363/168159103-0f97d8ec-1c5b-4fc8-8477-f704d2da5a0b.png)
![Screen Shot 2022-05-12 at 1 07 07 PM](https://user-images.githubusercontent.com/105457363/168159361-00528dc1-4c5e-44de-b497-4cbda79d0027.png)
![Screen Shot 2022-05-12 at 12 27 28 PM](https://user-images.githubusercontent.com/105457363/168159439-0948368d-2e53-4f19-ae01-0d46a21ba57c.png)
![Screen Shot 2022-05-12 at 12 28 34 PM](https://user-images.githubusercontent.com/105457363/168159468-e06837cf-a8f4-44e3-870e-f9e9a0f73601.png)
![Screen Shot 2022-05-12 at 12 28 38 PM](https://user-images.githubusercontent.com/105457363/168159476-edc179f7-063b-4886-83f2-5fea2a2cbd5f.png)

**Desktop (please complete the following information):**
 - Browser [chrome]

**Additional context**

**Comment by kasperbolarsen** [comment]
Have you verified that the Managed Properties / refinablestrings contains a value using SP Query Tool , SP Editor or similar tool?
